api/src/services/dataset.js

```diff
@@ -100,7 +101,7 @@ async function get_dataset({
   bundle = false,
   includeProjects = false,
   initiator = false,
-  include_upload_log = false,
```

Comment:
Upload log is now returned as part of the audit_logs that were already being returned by this API.